app/iht/views/application/declaration/probate_application_form_details.scala.html

```diff
@@ -34,11 +34,11 @@
 
     <p><a id="download-and-print" class="u--no-print" href="javascript:window.print()">@Messages("page.iht.application.probate.print")</a></p>
 
-    <div class="panel-indent">
-    <p>@Messages("page.iht.application.probate.guidance.indent")</p>
-    </div>
-
-    <p>@Messages("page.iht.application.probate.guidance.p2")</p>
+    <p>
+        <div class="panel-indent">
+        <p>@Messages("page.iht.application.probate.guidance.indent")</p>
+        </div>
+    </p>
 
     <a id="continue-to-estate-reports" class="button u--no-print" href="@iht.controllers.estateReports.routes.YourEstateReportsController.onPageLoad">
          @Messages("site.button.continue.iht.app.page")
```

Comment:
The text on this button should be "Go to your Inheritance Tax estate reports"

Also I think the first paragraph needs an {0} in it before the word "estate"